integration/benchmark/compare_results.jq

```diff
@@ -0,0 +1,32 @@
+# extrace the single specified benchmark results by name
```

Comment:
This could be my preference, but I'm not good at writing/maintaining jq. How about using Python or Go?

Reply:
@sbuckfelder suggested Go for this and the regression threshold steps. Pro for that approach is that containerd devs all(?) know(?) Go. Con is that it adds a compile step to the comparison.

Python keeps us in an interpreted language, and would probably be one of the most concise languages to describe this other than jq. I am unsure how familiar the containerd devs tend to be with Python.

Javascript and perl are also options, although not my preference. I rejected bash because it's very bad at handling JSON.